language-server/language-server-api/.js/src/main/scala/aqua/lsp/TypeShow.scala

```diff
@@ -0,0 +1,51 @@
+package aqua.lsp
+
+import aqua.types.*
+
+import cats.Show
+import cats.syntax.show.*
+
+object TypeShow {
```

Comment:
Why not put it to `object Type`? It would be useful to pretty print types not only in LSP results